src/Angor/Client/Pages/Create.razor

```diff
@@ -724,6 +724,9 @@
 
         try
         {
+            // Ensure default images are set for Testnet
+            SetDefaultImagesForTestnet(project.Metadata);
```

Comment:
may I suggest we do that on startup?
also can we also do that for the presets? so a user just goes next next next and invest

Reply:
When you say startup, you mean when you go into the create-project there will be these links?
And also yeah we can do it for presets, should I make it random or a pre-determined?

Reply:
yeah random put some images on the correct boxes when we land on the page if it is testnet a user can always just change it